Modules/Quotas/src/Quotas.Application/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace Backbone.Modules.Quotas.Application.Extensions;
+
+public static class IServiceCollectionExtensions
+{
+    public static void AddApplication(this IServiceCollection services)
+    {
```

Comment:
I intentionally added the commented out lines so that you can already add them. They will be needed soon, and it's nice if the person who implements the first command/validator can just uncomment the code instead of having to look up what's needed.

Reply:
Oops, ok. Ty